### PR TITLE
docs update ig?

### PR DESCRIPTION
**Pull request by ninsaccplayzyt**
idk if this is gonna work or not so @virtual-designer i wanna know if it will or not

### CODE REVIEW

**Comment by virtual-designer** [comment]
It works, though how about rewording the first sentence:

> v9 is the current version of SudoBot.

**Comment by ninsaccplayzyt** [comment]
> It works, though how about rewording the first sentence:
> 
> > v9 is the current version of SudoBot.

Done, hopefully it updated here since I'm on mobile currently

**Review by virtual-designer — APPROVED** [review]
LGTM!
